challenges/validate_telephone_numbers.js

```diff
@@ -10,7 +10,11 @@ See Instructions below
 
 function telephoneCheck(str) {
   // Good luck!
+<<<<<<< HEAD
```

Comment:
looks like you got some leftover merge nonsense in there...  Also make sure to pull down new master....